bundle/config/mutator/initialize_urls.go

```diff
@@ -0,0 +1,62 @@
+package mutator
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+)
+
+type initializeUrls struct {
+	name string
+}
+
+// InitializeURLs makes sure the URL field of each resource is configured.
+// NOTE: since this depends on an extra API call, this mutator adds some extra
+// latency. As such, it should only be used when needed.
+// This URL field is used for the output of the 'bundle summary' CLI command.
+func InitializeURLs() bundle.Mutator {
+	return &initializeUrls{}
+}
+
+func (m *initializeUrls) Name() string {
+	return fmt.Sprintf("ConfigureURLs(%s)", m.name)
+}
+
+func (m *initializeUrls) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+	workspaceId, err := b.WorkspaceClient().CurrentWorkspaceID(ctx)
+	orgId := strconv.FormatInt(workspaceId, 10)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	configureForOrgId(b, orgId)
+	return nil
+}
+
+func configureForOrgId(b *bundle.Bundle, orgId string) {
+	urlPrefix := b.Config.Workspace.Host
```

Comment:
This property is not guaranteed to be present. Users may configure the workspace by setting only the profile, by specifying a profile flag, or in the case of Azure by specifying the Azure resource ID, etc.

It may also be configured in non-canonical form and already have the `?o=` suffix.

To avoid all of these issues, you can use the SDK's configuration object here:
```
b.WorkspaceClient().Config.CanonicalHostName()
```

This field _always_ includes only the URL scheme and host, nothing more.

Reply:
Thanks! Updated.